docs/src/modules/javascript/examples/test/eventsourced/shoppingcart.ts

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2019 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import * as proto from "../lib/generated/proto";
+import { replies } from "@lightbend/akkaserverless-javascript-sdk";
+
+// tag::entity-class[]
+import { EventSourcedEntity } from "@lightbend/akkaserverless-javascript-sdk";
+
+const entity: EventSourcedEntity = new EventSourcedEntity(
+    ["shoppingcart.proto", "domain.proto"],
+    "example.shoppingcart.ShoppingCartService",
+    {
+        persistenceId: "shopping-cart",
+        snapshotEvery: 100
+    }
+);
+// end::entity-class[]
```

Comment:
Examples shouldn't be copied here manually. Use the compiled and tested samples, which are copied into the examples automatically (the rsync in the Makefile).

Reply:
Yes, I was quite confused about the fact that some of the examples are coming from a "static" place, but some of them from the dynamically copied samples.

Since I was not sure why is it made like this, I've replicated the same behaviour of the JS examples.
So where the JS examples took the static path, I did the same for the TS ones.